photon-lib/py/photonlibpy/simulation/photonCameraSim.py

```diff
@@ -432,7 +432,9 @@ def distance(target: VisionTargetSim):
         )
 
     def submitProcessedFrame(
-        self, result: PhotonPipelineResult, receiveTimestamp: float | None
+        self,
+        result: PhotonPipelineResult,
+        receiveTimestamp_us: float = wpilib.Timer.getFPGATimestamp() * 1e6,
```

Comment:
When will this default argument be evaluated? And is it re-evaluated every time?

Reply:
Good catch. Fixed.